### PR TITLE
Remove 5f tap buffer extension during tumble

### PR DESCRIPTION
**Pull request by jobrien97**
Removes the training wheels

Tap buffer during tumble 10f -> 5f

### CODE REVIEW

File: fighters/common/src/general_statuses/mod.rs [hunk 1]
```diff
@@ -90,7 +90,7 @@ pub unsafe fn status_pre_DamageAir(fighter: &mut L2CFighterCommon) -> L2CValue {
 
 #[skyline::hook(replace = smash::lua2cpp::L2CFighterCommon_sub_DamageFlyCommon_init)]
 pub unsafe fn damage_fly_common_init(fighter: &mut L2CFighterCommon) {
-    ControlModule::set_command_life_extend(fighter.module_accessor, 5);
+    ControlModule::set_command_life_extend(fighter.module_accessor, 0);
```
Comment (WuBoytH) [inline fighters/common/src/general_statuses/mod.rs]: this line shouldn't even exist if you wanted to get rid of this

**Review by WuBoytH — CHANGES_REQUESTED** [review]

File: fighters/common/src/general_statuses/mod.rs (hunk 1 above)
Comment (jobrien97) [reply fighters/common/src/general_statuses/mod.rs]: u get a comment out, take it or leave it

**Review by WuBoytH — APPROVED** [review]
bringus

File: fighters/common/src/general_statuses/mod.rs (hunk 1 above)
Comment (O1yyy) [reply fighters/common/src/general_statuses/mod.rs]: lgtm

**Pull request merged**